socorro/external/boto/crashstorage.py

```diff
@@ -262,7 +256,7 @@ def get_dumps_as_files(self, crash_id):
         # convert our native memory dump mapping into a file dump mapping.
         return in_memory_dumps.as_file_dumps_mapping(
             crash_id,
-            self.config.temporary_file_system_storage_path,
```

Comment:
One of the effects of this configuration variable and its use is that all thread stored the dump files of crash reports they were working on in the same directory with no uniqueness to the filename. If two threads happen to be working on the same crash report (this is possible--there's nothing to prevent it), then there's a race condition around deleting files and processing. That's the cause of bugs 1742100 and 1767282.